include/swift/ABI/MetadataValues.h

```diff
@@ -2249,17 +2255,17 @@ class TaskCreateFlags : public FlagSet<size_t> {
 public:
   enum {
     // Priority that user specified while creating the task
-    RequestedPriority = 0,
-    RequestedPriority_width = 8,
-
-    Task_IsChildTask                              = 8,
-    // Should only be set in task-to-thread model where Task.runInline is
-    // available
-    Task_IsInlineTask                             = 9,
-    Task_CopyTaskLocals                           = 10,
-    Task_InheritContext                           = 11,
-    Task_EnqueueJob                               = 12,
-    Task_AddPendingGroupTaskUnconditionally       = 13,
+    RequestedPriority                       = 0,
+    RequestedPriority_width                 = 8,
+
+    Task_IsChildTask                        = 8,
+    // Should only be set in task-to-thread model
+    // where Task.runInline is available
+    Task_IsInlineTask                       = 9,
+    Task_CopyTaskLocals                     = 10,
+    Task_InheritContext                     = 11,
+    Task_EnqueueJob                         = 12,
+    Task_AddPendingGroupTaskUnconditionally = 13, // used for: TaskGroup, TaskPool
```

Comment:
ABI: Can we rename this actually? I suspect no since ABI even though the value remains the same.